roles/artifacts/tasks/edpm.yml

```diff
@@ -74,6 +74,7 @@
           sudo test -d /var/lib/openstack && sudo cp -a /var/lib/openstack /tmp/{{ host_ip }}
           sudo test -d /var/lib/config-data && sudo cp -a /var/lib/config-data /tmp/{{ host_ip }}
           sudo test -d /var/lib/cloud && sudo cp -a /var/lib/cloud /tmp/{{ host_ip }}
+          sudo test -d /home/zuul/compliance-scans && sudo cp -a /home/zuul/compliance-scans /tmp/{{ host_ip }}
```

Comment:
Note for other reviewers: this is running against compute nodes. Due to the `run_once: true` associated to the compliance test, it would come from only one node.